modules/machine-user-infra-machines-pxe.adoc

```diff
@@ -63,10 +73,20 @@ This configuration does not enable serial console access on machines with a grap
 +
 [NOTE]
 ====
+ifndef::ibm-power[]
 To network boot the CoreOS `kernel` on `aarch64` architecture, you need to use a version of iPXE build with the `IMAGE_GZIP` option enabled. See link:https://ipxe.org/buildcfg/image_gzip[`IMAGE_GZIP` option in iPXE].
+endif::ibm-power[]
+ifdef::ibm-power[]
+To network boot the CoreOS `kernel` on `ppc64le` architecture, you need to use a version of iPXE build with the `IMAGE_GZIP` option enabled. See link:https://ipxe.org/buildcfg/image_gzip[`IMAGE_GZIP` option in iPXE].
+endif::ibm-power[]
 ====
 
+ifndef::ibm-power[]
 ** For PXE (with UEFI and GRUB as second stage) on `aarch64`:
+endif::ibm-power[]
+ifdef::ibm-power[]
+** For PXE (with UEFI and GRUB as second stage) on `ppc64le`:
+endif::ibm-power[]
```

Comment:
@prb112 
I have replaced `aarch64` with `ppc64le`. Request you to review if the steps hold good for `ppc64le`.

Reply:
Preview link: https://64357--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power#machine-user-infra-machines-pxe_creating-multi-arch-compute-nodes-ibm-power

Reply:
lgtm
/ack